packages/calypso-e2e/src/lib/pages/plans-page.ts

```diff
@@ -51,6 +52,11 @@ export class PlansPage {
 	 */
 	constructor( page: Page ) {
 		this.page = page;
+		page.on( 'response', ( res ) => {
+			if ( res.url().includes( 'active-promotions' ) ) {
+				this.receivedActivePromotions = true;
+			}
+		} );
```

Comment:
FYI, this enables the interception permanently until the browser is shut down IIRC.